### ISSUE TITLE
Hit targets fold over

### ISSUE THREAD

**Issue by arthurkehrwald** [freezy/VisualPinball.Engine#479]
https://github.com/user-attachments/assets/09932b93-4a11-49d3-b914-6b7f99e3a0e3

Hit targets rotate 90 degrees backwards when entering play mode and do not collide with the ball. Ticking the "Show Colliders" option in the ´HitTargetCollider` inspector does not show anything.

**Comment by freezy** [comment]
Yes, those definitely need more love. They also need to be properly scaled. I'll take care of that when doing the translation refactoring.

**Comment by freezy** [comment]
Should be fixed since the last big merge.
